public/sass/pages/_explore.scss

```diff
@@ -77,6 +77,26 @@
   }
 }
 
+.explore-container-content-item {
```

Comment:
As we are trying to migrate from css to `emotion` I think we should not be creating new classes if we can avoid it.

Reply:
Good catch. I'll rewrite it with `emotion` then.